### PR TITLE
Add nutanix realtime e2e test

### PR DESCRIPTION
**Pull request by sgaoshang**
Add nutanix realtime e2e test

### CODE REVIEW

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-nutanix-ipi-boot-categories-project-f28

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-nutanix-ipi-boot-categories-project-f28

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-nutanix-ipi-boot-categories-project-f28

**Comment by sgaoshang** [comment]
@gpei @jianlinliu pls help to review nutanix realtime e2e PR, thanks.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml [hunk 1]
```diff
@@ -1590,6 +1590,7 @@ tests:
     env:
       E2E_RUN_TAGS: '@nutanix-ipi'
       FIPS_ENABLED: "true"
+      RT_ENABLED: "true"
```
Comment (gpei) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml:1593]: please also add `-rt-` keyword to the job name to show it's a realtime kernel test

Comment (sgaoshang) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml:1593]: fixed, thanks.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml [hunk 2]
```diff
@@ -2438,14 +2438,15 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-ui
     workflow: cucushift-installer-rehearse-ibmcloud-ipi-workers-rhel8
-- as: nutanix-ipi-boot-categories-project-f28
+- as: nutanix-ipi-boot-categories-project-rt-f28
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:2441]: Suggest to enable rt on the other destructive ci job as a pair.

Comment (sgaoshang) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:2441]: Added it.

**Comment by jianlinliu** [comment]
cc @liangxia to review.

**Pull request merged**